tutorials/radiant-mlhub-on-demand-training-data.ipynb

```diff
@@ -0,0 +1,1281 @@
+   "source": [
+    "l8_original = stack(\n",
+    "    items=ItemCollection([best_l8_match]), assets=LANDSAT_8_RGB_BANDS, resolution=10\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aa276eef-73ea-4834-a6a1-7cbd205f3d46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_original"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd20472b-be7e-481a-b1ed-3bbcff044c7f",
+   "metadata": {},
+   "source": [
+    "As we can see from the metadata for the Xarray above, the Landsat 8 scene has a significantly larger geographic footprint, `~20,000 x ~20,000 pixels`, compared to `120 x 120 pixels` for the Sentinel-2 chips that were prepared for the dataset. We need to crop/mask the image down so they represent the exact same terrain.\n",
+    "\n",
+    "Luckily, the `bounds_latlon` parameter of `stackstac` makes it easy to crop the image to this size automatically for all bands/assets requested."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0045f54d-5076-4a14-9292-3dbdde339cdc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped = stack(\n",
+    "    items=ItemCollection([best_l8_match]),\n",
+    "    assets=LANDSAT_8_RGB_BANDS,\n",
+    "    bounds_latlon=source_item.bbox,\n",
+    "    resolution=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0f0600e6-3bb8-4da9-90b2-ee4ec45ea227",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7a5e313-0a2f-410e-8339-9630ca87dd78",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped[0].plot(col=\"band\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc46c85b-daa9-44df-9be7-62dfa1234b25",
+   "metadata": {},
+   "source": [
+    "Now we have a cropped Landsat 8 chip that spatially and temporally matches our Sentinel-2 source imagery and label sample from the BigEarthNet dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a70b063-d273-4aee-864b-07e318388890",
+   "metadata": {},
+   "source": [
+    "### Launch a Dask gateway cluster for parallel processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb650d80-bf1a-4aad-8f8b-08a612e28aae",
+   "metadata": {},
+   "source": [
+    "We will use Dask to optimize our data processing of hundreds of Landsat-8 scenes by parallelizing the workflow with a delayed computation graph. The Dask Client schedules, runs the delayed computations, and gathers the results, while the Dask Gateway provides a secure and centralized way of managing the multiple client clusters. This is especially useful for running Dask on Planetary Computer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29531759-6d19-4010-8401-eb947a32c515",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = dd_client(\n",
```

Comment:
Just use `dask.distributed.Client()` (or a GatewayCluster & get_client if doing this on a distributed cluster)

Reply:
for both `ps_client` and `dd_client` I simplified by removing the import alias, so users will directly use `distributed.Client` and `pystac_client.Client`